### PR TITLE
Updated docs for public access on paid projects

### PR DESCRIPTION
**Pull request by Aaen**

### CODE REVIEW

File: Umbraco-Cloud/Set-Up/Project-settings/index.md [hunk 1]
```diff
@@ -36,6 +36,8 @@ All Staging and Development environments on Umbraco Cloud projects are by defaul
 
 :::note
 On **Trial** projects the basic authentication is enabled on the Live environment. This cannot be removed before setting up a subscription for the Cloud project.
+
+On **Paid projects** you will have access to the pubic access feature when you are on the Standard plan or higher.
```
Comment (sofietoft) [inline Umbraco-Cloud/Set-Up/Project-settings/index.md:40]:
```suggestion
Access to disabling/enabling basic authentication requires your project to be on the Standard plan or higher. 

On **Trial** and **Starter** projects, the basic authentication is enabled on the Live environment and cannot be removed unless the project is upgraded to a higher plan.
```

**Review by sofietoft — COMMENTED** [review]
Good catch @Aaen !

I've made some small changes.
I think it makes sense to mention the plans where the feature is available first.
Let me know what you think!

File: Umbraco-Cloud/Set-Up/Project-settings/index.md (hunk 1 above)
Comment (Aaen) [reply Umbraco-Cloud/Set-Up/Project-settings/index.md:40]: Sounds good to me @sofietoft

**Comment by Aaen** [comment]
Now you should be able to merge it

**Pull request merged**